modules/pom.xml

```diff
@@ -35,6 +35,7 @@
         <module>schedulers</module>
         <module>scripting</module>
         <module>scripting-jruby</module>
+        <module>scripting-plugin</module>
```

Comment:
I misunderstood your previous comment. The plugin will be moved to a separated repo soon. Just created https://www.mulesoft.org/jira/browse/MULE-12420 to track it